install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -54,10 +54,29 @@ the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
 in {product-title} 3.9. The following examples show configuration files with the
 changed values.
 
-.Considerations when using MySQL and PostgreSQL with Azure file
-* The owner UID of the Azure File mounted directory is different from the UID of a container.
-* MySQL containers change the file owner permission in the mounted directory. Because of the mismatch between the owner UID and container process UID, this operation fails. Therefore to run MySQL with Azure File:
-** Specify the Azure File mounted directory UID in the `runAsUser` variable in the PV configuration file.
+.Considerations when using Azure File
```

Comment:
Lines 57 to 68 sound move like general restrictions than prereqs. I might move them to line 27.